### PR TITLE
Feature/add metadata to configuration

### PR DESCRIPTION
**Pull request by anitatomaszewska**
> Add metadata usable in the Magento to the Pimcore Integration configuration. 
Metadata added:
searchable
filterable
comparable
visible_on_front
used_in_product_listing

To each attribute table, attry_conf is added during mapping and send to the Magento

### CODE REVIEW

File: src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php [hunk 1]
```diff
@@ -51,6 +53,24 @@ protected function getFieldNames(Element $field, array $mappingArray)
...
+     * @param array $mappingArray
+     * @return array|mixed
+     */
+    protected function getAttrConf(Element $field, array $mappingArray)
+    {
+        $attrConf = [];
+        if (
+            array_key_exists($field->name, $mappingArray)
+            && array_key_exists(0, $mappingArray[$field->name])
+            && array_key_exists('attr_conf', $mappingArray[$field->name][0])
```
Comment (rpicheta) [inline src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php]: Use const instead string value.

File: src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php [hunk 2]
```diff
@@ -51,6 +53,24 @@ protected function getFieldNames(Element $field, array $mappingArray)
...
+        ) {
+            $attrConf = $mappingArray[$field->name][0]['attr_conf'];
```
Comment (rpicheta) [inline src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php]: Use const instead string value.

File: src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php (hunk 1 above)
Comment (anitatomaszewska) [reply src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php]: done

File: src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php (hunk 2 above)
Comment (anitatomaszewska) [reply src/Divante/MagentoIntegrationBundle/Application/Mapper/Strategy/AbstractMapStrategy.php]: done

**Comment by mbolka** [comment]
Thank you!

**Pull request merged**